20180405/MINUTES.md

```diff
@@ -0,0 +1,97 @@
+			* using py2exe, it could not work for project as big as Sardana, SOLARIS will check this
+	* MAX IV
+		* Measurement groups with faulty controllers are disappearing
+		* testing new releases of Sardana and Taurus - no complaints so far
+	* DESY
+		* Users would like to have motor limits checked before the scan (sardana-org/sardana#46)
+			* Motor limit checking won't allow scans to the limit
+			* Scans with custom unpredictible generators can not be checked
+			* Maybe we could check if the generateor "standard" and it only then check the limits
+			* Moved checks to pre-scan hook, so users can enable and disable it
+		* RunningMacros Door attribute
+			* possible enhancement, already used at DESY
+			* shows entire stack of currently running macros
+			* at SOLARIS we parse the Debug attribute for that
+			* Door's Status attribute should do the same thing
+3. Collaborative Developments - Review
+	* Jan18 review - TODOs left, bugs found
+		* -> slides
+		* Taurus 3 status
+			* DESY - soon on Debian 9 and Taurus 4
+			* MAX IV - deprecated
+			* SOLARIS - mostly deprecated, used exclusively for some legacy GUIs
+			* ALBA - still in use
+			* Conclusion: Taurus 3 won't get the FQDN update
+		* Bug distribution
+			* ALBA - Suse 11 testsuite problems
+			* SOLARIS - MacroEditor issues (if only MacroEditor will run)
+			* MAX IV - PreScanSnapshot issue
+			* DESY - problems related to Debian 9
+	* Spock logging - users feedback
+		* Users don't like the new solution
+		* Ideally, both old and new solution should be available, users could choose
+		* Exactly what appears on the screen should be logged (without "running macro", parentheses, etc.)
+		* There's a bug somewhere in the current implementation
+		* There's a Linux tool to create terminal session log, maybe it could work also for Spock
+4. New Developments, Enhancements or Bug Fixes
+	* Predefined values for string macro arguments
+		* tab autocompletion for strings in spock
+		* currently implemented in genutils, using MS environment variables "<macro name>+predefined"
+		* could be implemented using Sardana types
+		* PR pending
+	* dmesh and mesh macros
+		* external motors are moved for each point
+		* this can cause motor timeout
+		* users should be able to deactivate this
+		* complicated because of Motion objects
+		* send None as no movement (?)
+		* issue pending
+	* third-party repositories
+		* SEP 16 (https://github.com/sardana-org/sardana/pull/726)
+		* plugins more like Python packages -> would require massive changes to the plugin system
+		* Sardana organization would host only the 3rd party plugin register
+		* Everyone could use repositories layout of their choice
+		* Possible approach: single controller repos
+			* Pros:
+				* easier documentation
+				* allows better code structure
+				* fits better for CI
+			* Cons:
+				* you need to install multiple repos/packages instead of just one
+			* See https://github.com/amilan/lib-maxiv-sardana_albaem
```

Comment:
The link is pointing to the master branch, where unfortunately I haven't merge the refactoring branch, therefore, it's better to point to this url:
https://github.com/amilan/lib-maxiv-sardana_albaem/tree/refactoring

Reply:
Thanks for pointing this out. I'm changing ti.